hw3/index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="reset.css">
+	<link rel="stylesheet" type="text/css" href="style-hw3.css">
+	<link rel="stylesheet" type="text/css" href="response.css">
+</head>
+<body>
+	<header class="main-header">
+		<div class="logo">
+			<img src="image/logo.png" class="logo__img"> 
+			<div class="logo__name">Global</div>
+		</div>
+		<nav class="main-nav">
+			<ul class="main-nav__list">
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Home</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Portfolio</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">About</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Services</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Projects</a>
+				</li>
+				<li class="main-nav__item">
+					<a href="#" class="main-nav__link">Contact</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+	<div class="banner">
+		<div class="banner__images">
+			<img src="image/banner-1.png" alt="" class="banner__img banner__img-1">
+			<img src="image/banner-2.png" alt="" class="banner__img banner__img-2">
+			<img src="image/banner-3.png" alt="" class="banner__img banner__img-3">
+		</div>
+		<div class="banner__info">
+			<div class="banner-container">
+				<div class="banner__title">Lorest Nesto Gravido Forum</div>
+				<div class="banner__text">Duis porta ultrices elit sed consequat. Suspendisse nunc orci, molestie sed mollis nec, facilisis at 	ligula. Donec ultrices varius lacinia. Pellentesque id tortor non justo consequat pulvinar. </div>
+				<button type="submit" class="banner__button banner__button_button">See More</button>
+			</div>
+		</div>
+	</div>
+	<div class="subscription">
+		<div class="subscription__text">Duis porta ultrices elit sed consequat. Suspendisse nunc orci molestie sed.</div>
+		
+		<form class="subscription__email">
+			<input class="subscription__email" type="email" name="email" placeholder="Email">
+			<input class="subscription__submit subscription__submit_blue" type="submit" name="submit" value="Submit">
+		</form>
+	</div>
+	<div class="preview">
+		<div class="preview__center">
+			<section class="preview__item preview__item-top">
```

Comment:
use for modifiers preview__item--top or preview__item_top - it should be the same way across the project.